### PR TITLE
ras/rastools: Restructure tests to align tools with corresponding package

### PR DESCRIPTION
**Pull request by sacsant**
There are 3 different tests to cover various Linux on POWER RAS
tools. These tools are part of 3 different packages
powerpc-utils, ppc64-diag and lsvpd. Restructure the tests such
that all tools from one package can be covered by a single test.

[patch 1/5]ras/rastools: Rename tests as per packages
[patch 2/5] ras/rastools: Consolidate sub tests
[patch 3/5] ras/rastools: Update yaml files
[patch 4/5] ras/rastools: Code style fixes
[patch 5/5] ras/rastools: Code optimization and fixes

Signed-off-by: Sachin Sant <sachinp@linux.ibm.com>

### CODE REVIEW

**Comment by sacsant** [comment]
Runs from SLES:

avocado run --nrunner-max-parallel-tasks=1 ras/ras_lsvpd.py 
JOB ID     : 826704dd185085e835f23ac2affd8ca7fd29a465
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-10-08T01.37-826704d/job.log
(1/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_vpdupdate: STARTED
 (1/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_vpdupdate: PASS (0.99 s)
 (2/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvpd: STARTED
 (2/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvpd: PASS (48.96 s)
 (3/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lscfg: STARTED
 (3/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lscfg: PASS (0.71 s)
 (4/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsmcode: STARTED
 (4/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsmcode: PASS (0.60 s)
 (5/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvio: STARTED
 (5/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvio: PASS (0.49 s)
 (6/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_locking_mechanism: STARTED
 (6/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_locking_mechanism: FAIL: Database corruption detected (5.69 s)

avocado run --nrunner-max-parallel-tasks=1 ras/ras_ppcdiag.py
JOB ID     : ed1281d52f42f6e5a9f7cbb426a8928aa1802812
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-10-08T01.41-ed1281d/job.log
 (1/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_nvsetenv: STARTED
 (1/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_nvsetenv: PASS (0.56 s)
 (2/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysattn: STARTED
 (2/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysattn: PASS (0.56 s)
 (3/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysfault: STARTED
 (3/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysfault: PASS (0.52 s)
 (4/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysident: STARTED
 (4/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysident: CANCEL: The identify indicators are not supported on this system (0.53 s)

avocado run --nrunner-max-parallel-tasks=1 ras/ras_ppcutils.py
JOB ID     : 857370b36c4609b4f361ad48d097bc4466ad62d2
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-10-08T01.57-857370b/job.log
 (01/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_set_poweron_time: STARTED
 (01/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_set_poweron_time: PASS (0.51 s)
 (02/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_sys_ident_tool: STARTED
 (02/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_sys_ident_tool: PASS (0.59 s)
 (03/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_drmgr: STARTED
 (03/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_drmgr: PASS (3.58 s)
 (04/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsprop: STARTED
 (04/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsprop: PASS (0.52 s)
 (05/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsslot: STARTED
 (05/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsslot: PASS (0.64 s)
 (06/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_nvram: STARTED
 (06/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_nvram: PASS (0.52 s)
 (07/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ofpathname: STARTED
 (07/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ofpathname: PASS (0.58 s)
 (08/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_ibm_get_vpd: STARTED
 (08/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_ibm_get_vpd: PASS (0.54 s)
 (09/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_errd_and_rtas_dump: STARTED
 (09/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_errd_and_rtas_dump: PASS (0.85 s)
 (10/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_event_decode: STARTED
 (10/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_event_decode: PASS (0.56 s)
 (11/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_uesensor: STARTED
 (11/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_uesensor: PASS (0.56 s)
 (12/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_serv_config: STARTED
 (12/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_serv_config: PASS (0.60 s)
 (13/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vscsi: STARTED
 (13/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vscsi: PASS (0.51 s)
 (14/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_veth: STARTED
 (14/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_veth: PASS (0.52 s)
 (15/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vdev: STARTED
 (15/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vdev: PASS (0.49 s)
 (16/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsdevinfo: STARTED
 (16/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsdevinfo: PASS (1.08 s)
 (17/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_hvcsadmin: STARTED
 (17/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_hvcsadmin: PASS (0.58 s)
 (18/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_bootlist: STARTED
 (18/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_bootlist: PASS (0.86 s)

**Comment by sacsant** [comment]
Runs from RHEL

avocado run --nrunner-max-parallel-tasks=1 ras/ras_lsvpd.py 
JOB ID     : 67c17dff93bc3673a355d294a8e6743642b78aeb
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-10-08T01.37-67c17df/job.log
(1/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_vpdupdate: STARTED
 (1/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_vpdupdate: PASS (1.33 s)
 (2/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvpd: STARTED
 (2/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvpd: PASS (43.26 s)
 (3/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lscfg: STARTED
 (3/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lscfg: PASS (0.84 s)
 (4/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsmcode: STARTED
 (4/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsmcode: PASS (0.90 s)
 (5/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvio: STARTED
 (5/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvio: PASS (0.73 s)
 (6/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_locking_mechanism: STARTED
 (6/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_locking_mechanism: PASS (2.86 s)

avocado run --nrunner-max-parallel-tasks=1 ras/ras_ppcdiag.py
JOB ID     : b7fa0c0987654fa620daf3708d8cc90ef3eddba2
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-10-08T01.51-b7fa0c0/job.log
 (1/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_nvsetenv: STARTED
 (1/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_nvsetenv: PASS (0.80 s)
 (2/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysattn: STARTED
 (2/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysattn: PASS (0.75 s)
 (3/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysfault: STARTED
 (3/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysfault: PASS (0.77 s)
 (4/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysident: STARTED
 (4/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysident: FAIL: ['usysident -l U78D8.ND0.FGD003M -s normal'] command(s) failed to execute   (1.01 s)

avocado run --nrunner-max-parallel-tasks=1 ras/ras_ppcutils.py
JOB ID     : 634ff5d41c4d99dc550544eb5b38c8c645fb507c
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2022-10-08T01.58-634ff5d/job.log
 (01/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_set_poweron_time: STARTED
 (01/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_set_poweron_time: PASS (0.71 s)
 (02/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_sys_ident_tool: STARTED
 (02/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_sys_ident_tool: PASS (0.76 s)
 (03/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_drmgr: STARTED
 (03/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_drmgr: PASS (4.59 s)
 (04/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsprop: STARTED
 (04/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsprop: PASS (0.97 s)
 (05/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsslot: STARTED
 (05/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsslot: FAIL: RAS: Failed commands are: ['lsslot -c pci -o'] (0.95 s)
 (06/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_nvram: STARTED
 (06/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_nvram: PASS (0.68 s)
 (07/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ofpathname: STARTED
 (07/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ofpathname: PASS (0.81 s)
 (08/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_ibm_get_vpd: STARTED
 (08/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_ibm_get_vpd: PASS (0.76 s)
 (09/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_errd_and_rtas_dump: STARTED
 (09/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_errd_and_rtas_dump: PASS (1.03 s)
 (10/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_event_decode: STARTED
 (10/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_event_decode: PASS (0.67 s)
 (11/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_uesensor: STARTED
 (11/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_uesensor: PASS (0.70 s)
 (12/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_serv_config: STARTED
 (12/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_serv_config: PASS (0.90 s)
 (13/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vscsi: STARTED
 (13/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vscsi: PASS (0.68 s)
 (14/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_veth: STARTED
 (14/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_veth: PASS (0.67 s)
 (15/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vdev: STARTED
 (15/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vdev: PASS (0.75 s)
 (16/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsdevinfo: STARTED
 (16/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsdevinfo: PASS (1.31 s)
 (17/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_hvcsadmin: STARTED
 (17/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_hvcsadmin: PASS (0.87 s)
 (18/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_bootlist: STARTED
 (18/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_bootlist: PASS (1.08 s)

**Comment by sacsant** [comment]
Runs from Ubuntu:

avocado run --nrunner-max-parallel-tasks=1 ras/ras_lsvpd.py 
JOB ID     : 3d737f03a4331a6143a017a19a66790efe32063d
JOB LOG    : /home/sachin/avocado-fvt-wrapper/results/job-2022-10-08T06.39-3d737f0/job.log
 (1/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_vpdupdate: STARTED
 (1/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_vpdupdate: PASS (0.92 s)
 (2/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvpd: STARTED
 (2/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvpd: PASS (57.83 s)
 (3/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lscfg: STARTED
 (3/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lscfg: PASS (0.43 s)
 (4/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsmcode: STARTED
 (4/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsmcode: PASS (0.41 s)
 (5/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvio: STARTED
 (5/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_lsvio: PASS (0.27 s)
 (6/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_locking_mechanism: STARTED
 (6/6) ras/ras_lsvpd.py:RASToolsLsvpd.test_locking_mechanism: FAIL: Database corruption detected (5.65 s)

avocado run --nrunner-max-parallel-tasks=1 ras/ras_ppcdiag.py
JOB ID     : 5c9e6205f0e88834395dc16f2afac66f283fa055
JOB LOG    : /home/sachin/avocado-fvt-wrapper/results/job-2022-10-08T06.57-5c9e620/job.log
 (1/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_nvsetenv: STARTED
 (1/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_nvsetenv: PASS (0.27 s)
 (2/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysattn: STARTED
 (2/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysattn: PASS (0.30 s)
 (3/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysfault: STARTED
 (3/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysfault: PASS (0.30 s)
 (4/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysident: STARTED
 (4/4) ras/ras_ppcdiag.py:RASToolsPpcdiag.test_usysident: CANCEL: The identify indicators are not supported on this system (0.27 s)

avocado run --nrunner-max-parallel-tasks=1 ras/ras_ppcutils.py
JOB ID     : 4626fba00364f2f5bd0f2aa34de719c8bd3ac74d
JOB LOG    : /home/sachin/avocado-fvt-wrapper/results/job-2022-10-08T06.59-4626fba/job.log
 (01/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_set_poweron_time: STARTED
 (01/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_set_poweron_time: PASS (0.28 s)
 (02/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_sys_ident_tool: STARTED
 (02/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_sys_ident_tool: PASS (0.26 s)
 (03/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_drmgr: STARTED
 (03/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_drmgr: PASS (1.89 s)
 (04/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsprop: STARTED
 (04/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsprop: PASS (0.25 s)
 (05/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsslot: STARTED
 (05/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsslot: PASS (0.40 s)
 (06/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_nvram: STARTED
 (06/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_nvram: PASS (0.27 s)
 (07/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ofpathname: STARTED
 (07/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ofpathname: PASS (0.38 s)
 (08/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_ibm_get_vpd: STARTED
 (08/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_ibm_get_vpd: PASS (0.30 s)
 (09/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_errd_and_rtas_dump: STARTED
 (09/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_errd_and_rtas_dump: PASS (0.47 s)
 (10/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_event_decode: STARTED
 (10/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_rtas_event_decode: PASS (0.29 s)
 (11/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_uesensor: STARTED
 (11/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_uesensor: PASS (0.27 s)
 (12/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_serv_config: STARTED
 (12/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_serv_config: PASS (0.34 s)
 (13/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vscsi: STARTED
 (13/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vscsi: PASS (0.27 s)
 (14/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_veth: STARTED
 (14/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_veth: PASS (0.28 s)
 (15/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vdev: STARTED
 (15/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_ls_vdev: PASS (0.27 s)
 (16/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsdevinfo: STARTED
 (16/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_lsdevinfo: PASS (0.83 s)
 (17/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_hvcsadmin: STARTED
 (17/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_hvcsadmin: PASS (0.40 s)
 (18/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_bootlist: STARTED
 (18/18) ras/ras_ppcutils.py:RASToolsPpcutils.test_bootlist: PASS (0.83 s)

File: ras/ras_lsvpd.py [hunk 1]
```diff
@@ -25,23 +26,30 @@
...
+    package/repository
+
+    :avocado: tags=ras,ppc64le
     """
     is_fail = 0
 
     def run_cmd(self, cmd):
-        if (process.run(cmd, ignore_status=True, sudo=True, shell=True)).exit_status:
+        if (process.run(cmd, ignore_status=True, sudo=True,
+                        shell=True)).exit_status:
```
[the remark this answers is not in the document]
Comment (sacsant) [reply ras/ras_lsvpd.py:41]: I haven't changed the logic in my patch. This is the original code. I believe the return code is still checked only an exception is not raised on failure.

File: ras/ras_lsvpd.py [hunk 2]
```diff
@@ -52,119 +60,11 @@ def run_cmd_out(cmd):
...
-                    (disk_name, file_path), ignore_status=True, sudo=True, shell=True)
-        process.run("echo %s >> %s" %
-                    (interface, file_path), ignore_status=True, sudo=True, shell=True)
-        self.run_cmd("bootlist -r -m both -f %s" % file_path)
-        if self.is_fail >= 1:
-            self.fail("%s command(s) failed in bootlist tool "
-                      "verification" % self.is_fail)
-
     @skipIf(IS_KVM_GUEST, "This test is not supported on KVM guest platform")
-    def test1_vpdupdate(self):
```
[the remark this answers is not in the document]
Comment (sacsant) [reply ras/ras_lsvpd.py:167]: These sub-tests are removed from this test and added to ras_ppcutils.py test (as these tools are part of powerpc-utils package/repository)

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Pull request merged**